src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5256,4 +5259,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InDynamicMethodArg" xml:space="preserve">
     <value>Arguments with 'in' modifier cannot be used in dynamically dispatched expessions.</value>
   </data>
+  <data name="ERR_TupleSizesMismatchForBinOps" xml:space="preserve">
+    <value>Tuple types used as operands of a binary operator must have matching cardinalities. But this operator has tuple types of cardinality {0} on the left and {1} on the right.</value>
```

Comment:
>binary [](start = 45, length = 6)

Should we be more specific and say equality here instead?

Reply:
I don't feel strongly about this. The trouble is to concisely describe both `==` and `!=`. 
I'm concerned that saying "equality operator" can be ambiguous (does that mean just `==` or both `==` and `!=` ?).

---
In reply to: [170485560](https://github.com/dotnet/roslyn/pull/23356#discussion_r170485560) [](ancestors = 170485560)

Reply:
> I'm concerned that saying "equality operator" can be ambiguous (does that mean just == or both == and != ?).

We can say "equality/inequality",or "==/!=".

---
In reply to: [170739547](https://github.com/dotnet/roslyn/pull/23356#discussion_r170739547) [](ancestors = 170739547,170485560)